### PR TITLE
Table Tests

### PR DESCRIPTION
**Pull request by joshreisner**
```
  Table.jsx                |   97.72 |      100 |   92.85 |   97.67 | 173
```

Line 173 is the infinite scrolling, which I couldn't figure out how to mock.

This PR refactors the "meetings in progress" banner to use a button instead of a link.

Closes #211

### CODE REVIEW

File: src/components/Table.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        state={mockState}
+        setState={jest.fn()}
+        filteredSlugs={filteredSlugs}
+        inProgress={[]}
+        listButtons={false}
+      />
+    );
+    const anchors = screen.getAllByRole('link');
+    expect(anchors).toHaveLength(2);
+    anchors.forEach(anchor => fireEvent.click(anchor));
```
Comment (altruisticsoftware) [inline src/components/Table.spec.jsx:57]: Do these click events buy us anything if we're not making assertions? Perhaps we could just check the `href`s?

File: src/style.scss [hunk 2]
```diff
@@ -269,29 +269,6 @@ div.tsml-ui {
       border-bottom: 1px solid $gray-400;
     }
 
-    tbody {
```
Comment (altruisticsoftware) [inline src/style.scss:272]: Love seeing css go away 😍

File: src/components/Table.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        filteredSlugs={filteredSlugs}
+        inProgress={['foo', 'bar']}
+        listButtons={true}
+      />
+    );
+    const anchors = screen.getAllByRole('link');
+    expect(anchors).toHaveLength(5);
+
+    const button = screen.getByRole('button');
+    fireEvent.click(button);
```
Comment (altruisticsoftware) [inline src/components/Table.spec.jsx:89]: Any assertions we could make here? 👀

**Review by altruisticsoftware — COMMENTED** [review]
Woop woop 💥

File: src/components/Table.spec.jsx (hunk 1 above)
Comment (joshreisner) [reply src/components/Table.spec.jsx:57]: yeah good point i was trying to buy some line coverage for cheap :)

i'll take another look tomorrow

Comment (joshreisner) [reply src/components/Table.spec.jsx]: ok better now!

Comment (altruisticsoftware) [reply src/components/Table.spec.jsx]: Heck yeah 😎

**Review by altruisticsoftware — APPROVED** [review]

**Pull request merged**